nodejs/package.json

```diff
@@ -25,22 +25,22 @@
   },
   "homepage": "https://github.com/slackapi/tasks-app#readme",
   "dependencies": {
-    "@slack/bolt": "^3.6.0",
+    "@slack/bolt": "^3.8.1",
     "dotenv": "^10.0.0",
     "eslint-plugin-jest": "^24.4.2",
     "luxon": "^1.28.0",
     "nodemon": "^2.0.12",
     "pluralize": "^8.0.0",
     "sequelize": "^6.6.5",
     "slack-block-builder": "^2.1.1",
-    "sqlite3": "^5.0.2"
```

Comment:
Are we 100% this doesn't break anything? The major semver gives me pause, but I appreciate we barely scratch the surface of DB operations.

Reply:
@colmdoyle I don't know why `npm audit fix` did this but will check later

Reply:
@colmdoyle 
Somehow, sqlite3@5 depends on a very old version range of node-gyp (3.x vs the latest is v8). 
```
$ npm audit
# npm audit report

tar  <=4.4.17
Severity: high
Arbitrary File Creation/Overwrite on Windows via insufficient relative path sanitization - https://github.com/advisories/GHSA-5955-9wpr-37jh
Arbitrary File Creation/Overwrite via insufficient symlink protection due to directory cache poisoning using symbolic links - https://github.com/advisories/GHSA-9r2w-394v-53qc
Arbitrary File Creation/Overwrite due to insufficient absolute path sanitization - https://github.com/advisories/GHSA-3jfq-g458-7qm9
Arbitrary File Creation/Overwrite via insufficient symlink protection due to directory cache poisoning - https://github.com/advisories/GHSA-r628-mhmh-qjhw
fix available via `npm audit fix --force`
Will install sqlite3@4.2.0, which is a breaking change
node_modules/sqlite3/node_modules/tar
  node-gyp  <=3.8.0
  Depends on vulnerable versions of tar
  node_modules/sqlite3/node_modules/node-gyp
    sqlite3  >=5.0.0
    Depends on vulnerable versions of node-gyp
    node_modules/sqlite3

3 high severity vulnerabilities
```

Perhaps, there is something wrong with the sqlite3 npm package distribution in the latest major version. That's why `npm audit fix --force` downgraded the major to the stable v4.2. This should be reasonable for now. What do you think?

Reply:
Yeah the `sqlite3` package has a critical vulnerability they haven't patched.

I just checked out your branch and did a bit of manual testing and everything seems ok, so I say we merge.